README.md

```diff
@@ -12,6 +12,7 @@ Available addons
 addon | version | summary
 --- | --- | ---
 [product_brand](product_brand/) | 9.0.1.0.0 | Product Brand Manager
+[product_manufacturer](product_manufacturer/) | 9.0.1.0.0 | Products Manufacturers
```

Comment:
Don't change this file. It's automatically changed when modules get merged, and it will produce conflicts, so please clean your PR from this modification.